### PR TITLE
feat(dashboard-refresher): add refresh event to refresher

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
- add interval refresh feature
- add Immediately refresh feature

https://user-images.githubusercontent.com/50662170/208659776-fe85a878-d30b-40cd-a0a0-73ea0652264e.mov

(I modified some code for the test.)

### CODE REVIEW

File: src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue [hunk 1]
```diff
@@ -96,7 +96,8 @@ const fetchData = async () => new Promise((resolve) => {
 
 const drawChart = (chartData) => {
     const { chart, xAxis, yAxis } = createXYVerticalChart();
-    setChartColors(chart, state.colorSet);
+    // TODO: Temporary annotation processing for refresh implementation
+    // setChartColors(chart, state.colorSet);
```
Comment (sulmoJ) [inline src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue:100]: When using the AWS CloudFrontCost widget, colorSet causes a call stack overflow.
First of all, I processed the annotation to implement the Dashboard Refresh. 🙏 🙏 

image

File: src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue [hunk 2]
```diff
@@ -152,7 +153,7 @@ useWidgetLifecycle({
     initWidget, disposeWidget: disposeRoot,
 });
 
-defineExpose({
+defineExpose<{refreshWidget:() => void}>({
```
Comment (WANZARGEN) [inline src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue:156]:
```suggestion
defineExpose<{refreshWidget:() =>Promise<void>}>({
```

**Review by WANZARGEN — APPROVED** [review]
LGTM~ I left minor comment!!

File: src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue (hunk 1 above)
Comment (yuda110) [reply src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue:100]: Oops...I'll check it

**Review by yuda110 — APPROVED** [review]
LGTM!

File: src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue (hunk 1 above)
Comment (sulmoJ) [reply src/services/dashboards/widgets/aws-cloud-front-cost/AWSCloudFrontCost.vue:100]: 🙏 🙏 🙏

**Pull request merged**